MAUI/TimePicker/getting-started.md

```diff
@@ -0,0 +1,189 @@
+
+3. To initialize the control, import the control namespace `Syncfusion.Maui.Picker` in XAML or C# code.
+
+4. Initialize `SfTimePicker.`
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3 5" %}
+
+<ContentPage   
+    . . .
+    xmlns:picker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker">
+
+    <picker:SfTimePicker />
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="1 9 10" %}
+
+using Syncfusion.Maui.Picker;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfTimePicker picker = new SfTimePicker();
+        this.Content = picker;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+using Syncfusion.Maui.Core.Hosting;
+namespace GettingStarted
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+
+            builder.ConfigureSyncfusionCore();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set header to the Time Picker
+
+The SfTimePicker control allows you to add the header text by setting the [Text] property in the [PickerHeaderView]. To enable the header view by setting the [Height] property in [PickerHeaderView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3" %}
+
+<picker:SfTimePicker x:Name="picker">
+    <picker:SfTimePicker.HeaderView>
+        <picker:PickerHeaderView Text="Time Picker" Height="40" />
+    </picker:SfTimePicker.HeaderView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="4 5" %}
+
+SfTimePicker picker = new SfTimePicker();
+picker.HeaderView = new PickerHeaderView()
+{
+    Text = "Time Picker",
+    Height = 40,
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set footer to the Time Picker
+
+In SfTimePicker control, validation buttons (OK and Cancel) can be customized by setting the [OkButtonText] and [CancelButtonText] property in the [PickerFooterView]. It allows you to confirm or cancel the selected time.  The `OkButtonText` can be enabled by using the [ShowOkButton] property in [PickerFooterView].
```

Comment:
In the SfTimePicker control, validation buttons (OK and Cancel) can be customized by setting the [OkButtonText] and [CancelButtonText] properties in the [PickerFooterView]. It allows you to confirm or cancel the selected time.  The `OkButtonText` can be enabled using the [ShowOkButton] property in the [PickerFooterView].

Reply:
Content modified.